cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -178,8 +178,42 @@ Docker and CDAP Applications
 
 .. include:: ../dev-env.rst  
    :start-line: 7
+   :end-line: 23
 
-.. include:: ../start-stop-cdap.rst  
-   :start-line: 4
+.. tabbed-parsed-literal::
+   :tabs: "Linux Virtual Machine"
+   :independent:
+
+   $ mvn archetype:generate \
+       -DarchetypeGroupId=co.cask.cdap \
+       -DarchetypeArtifactId=cdap-app-archetype \
+       -DarchetypeVersion=\ |release|
+
+.. include:: ../dev-env.rst  
+   :start-line: 30
+
+Starting and Stopping Standalone CDAP
+============================================
+
+Use the ``cdap.sh`` script to start and stop the Standalone CDAP:
+
+.. tabbed-parsed-literal::
+   :tabs: "Linux Virtual Machine"
+   :independent:
+
+   $ cd /opt/cdap/sdk
+   $ ./bin/cdap.sh start
+     . . .
+   $ ./bin/cdap.sh stop
+
```

Comment:
This is noted on the following line, but we can move it up. These are here so that people know how to stop (and restart) the SDK as required, especially as they need to if they are following some examples.